frontend/src/bundles/landing/components/button-tabs/styles.module.scss

```diff
@@ -0,0 +1,48 @@
+.tabs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: fit-content;
+    height: 36px;
+    margin: 0;
+    padding: 6px;
+    font-family: var(--font-family);
+    line-height: 20px;
+    list-style: none;
+    background-color: #eff3ff;
+    border-radius: 8px;
+}
+
+.tab {
+    display: flex;
+    align-items: center;
+    height: 24px;
+    padding: 0 25px;
+    color: var(--color-blue-600);
+    font-weight: var(--font-weight-600);
+    text-decoration: none;
+    background-color: transparent;
+    border: none;
+}
+
+.tab:hover {
+    cursor: default;
+}
+
+.tabs:last-child {
+    margin-right: 0;
+}
+
+.tab.active {
```

Comment:
```suggestion
.activeTab {
```

Reply:
Resolved in [834a319](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/834a319e247b1efe74173fcd33bc96a78504a5aa)